src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -226,10 +226,15 @@ static bool addTriggeredSubscriptions
   auto_ptr<DBClientCursor> cursor;
   BSONObj                  query = BSON("$or" << BSON_ARRAY(queryNoPattern.obj() << queryPattern.obj()));
 
-  if (!collectionQuery(getSubscribeContextAvailabilityCollectionName(tenant), query, &cursor, &err))
+  TIME_STAT_MONGO_READ_WAIT_START();
+  DBClientBase* connection = getMongoConnection();
+  if (!collectionQuery(connection, getSubscribeContextAvailabilityCollectionName(tenant), query, &cursor, &err))
   {
+    TIME_STAT_MONGO_READ_WAIT_STOP();
+    releaseMongoConnection(connection);
     return false;
   }
+  TIME_STAT_MONGO_READ_WAIT_STOP();
```

Comment:
I have commented about this in stadistics.md (commit 5974d5f4658d64823329d6122c2a4a49fdfda636)